app/models/stripe_wrapper.rb

```diff
@@ -1,22 +1,65 @@
 module StripeWrapper
+  def self.set_api_key(key)
+    Stripe.api_key = key
```

Comment:
Well I have noticed you have set it in initializer and have used this method with wrapper. It is OK, however I see it like this first time.